engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/repository-details-manage-ostree-branches.controller.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.repositories.controller:RepositoryManageContentController
+ *
+ * @requires $scope
+ * @requires translate
+ * @requires Repository
+ *
+ * @description
+ *   Provides the functionality for the repository details pane.
+ */
+angular.module('Bastion.repositories').controller('RepositoryManageOstreeBranchesController',
+    ['$scope', 'translate', 'Repository',
+    function ($scope, translate, Repository) {
+
+        $scope.wrap = function (repository) {
+            var branches = repository["ostree_branches"];
+            if (angular.isUndefined(branches)) {
+                branches = [];
+            }
+            $scope.branchObjects = _.map(branches, function (branch) {
+                return {name: branch};
+            });
+        };
+
+        $scope.resetBranch = function () {
+            $scope.branch = {
+                editMode: false,
+                working: false
+            };
+        };
+
+        $scope.resetBranch();
+        $scope.repository = Repository.get({id: $scope.$stateParams.repositoryId});
+
+        $scope.repository.$promise.then($scope.wrap);
+
+        $scope.removeBranches = function () {
+            var branches, branchObjectsToRemove = _.reject($scope.branchObjects, function (branch) {
```

Comment:
I tend to prefer variables being assigned to something on their own line as that makes it clearer/cleaner.